dcos/dcos-ui
# DCOS-51826: In Service Create/Edit form, add asterisk to container image when docker is selected

Add asterisk for container image when docker is selected.

Closes https://jira.mesosphere.com/browse/DCOS-51826

## Testing
1. In Services tab, edit or create a single-container service.
2. In the first tab, click "More Settings" and select Docker as Container Runtime.
3. Verify that a red asterisk is shown next to "Container Image".
4. Verify that the asterisk disappears when UCR is selected as Container Runtime.

## Trade-offs
None.

## Dependencies
None.

## Screenshots
### Before
![Screenshot from 2019-04-15 16-17-44](https://user-images.githubusercontent.com/40791275/56135726-067d7b80-5f9a-11e9-8f1f-300bb82ba0bb.png)

### After
![Screenshot from 2019-04-15 16-09-15](https://user-images.githubusercontent.com/40791275/56135507-953dc880-5f99-11e9-95dc-9ec6ee01d7f8.png)
